jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -202,10 +205,12 @@ YUI.add('entity-details', function() {
 
     render: function() {
       return (
-        <div className={this._generateClasses()}>
+        <div className={this._generateClasses()}
+          ref="content"
+          tabIndex="0">
```

Comment:
@hatched Ah, no setting the tabIndex just allows it to be tabbed to. You can set the order, but we don't.